### PR TITLE
Update transform-object-assign's README from babel.github.io [skip ci]

### PR DESCRIPTION
**Pull request by lukyth**
| Q                 | A 
| ----------------- | ---
| Doc PR            | yes

Update `babel-plugin-transform-object-assign`'s README.md to match with its website's README, which is located at [https://github.com/babel/babel.github.io/blob/master/docs/plugins/transform-object-assign.md](https://github.com/babel/babel.github.io/blob/master/docs/plugins/transform-object-assign.md)

### CODE REVIEW

File: packages/babel-plugin-transform-object-assign/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # babel-plugin-transform-object-assign
 
-Replace `Object.assign` with an inline helper.
+Replace `Object.assign` with an inline helper. If you are authoring an application, rather than a library, it is recommended that you use the `Object.assign` polyfill instead.
```
Comment (existentialism) [inline packages/babel-plugin-transform-object-assign/README.md]: Small nit, but we've been standardizing on adding a `>` before the descriptions.

See: https://github.com/babel/babel/blob/master/packages/babel-preset-es2017/README.md

**Comment by lukyth** [comment]
@existentialism Fixed! 😄 

Did I do the `[skip ci]` right? It seems that the CI is still running. 🤔

**Comment by hzoo** [comment]
It's all good - circle ci runs on `[skip ci]` if you aren't a collaborator

**Comment by hzoo** [comment]
Nice thanks a lot

**Pull request merged**